src/sdk/storage/identityVerificationStorageManager.ts

```diff
@@ -71,11 +72,22 @@ export class IdentityVerificationStorageManager {
         const doc = await this.repository.findByIdAndType(sessionId, type);
 
         if (doc) {
-            doc.vc = vc.toString();
-            doc.status = status;
             return this.updateRecord(sessionId, type, status, vc);
         } else {
             return this.create(sessionId, type, status, vc);
         }
     }
+
+    async countReuseableLogin(type?: VerificationTypeEnum): Promise<number> {
```

Comment:
`async countReuse()` would be better as the data sharing is not related to the login